tests/e2e/initialization/config.go

```diff
@@ -52,24 +52,29 @@ const (
 	AtomDenom           = "uatom"
 	OsmoIBCDenom        = "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518"
 	StakeIBCDenom       = "ibc/C053D637CCA2A2BA030E2C5EE1B28A16F71CCB0E45E8BE52766DC1B241B7787"
+	UstIBCDenom         = "ibc/BE1BB42D4BE3C30D50B68D7C41DB4DFCE9678E8EF8C539F6E6A9345048894FCC"
```

Comment:
Hi @davidterpay . Please manually backport just this file to `v14.x` branch.

Once merged, please ping me, and I will give you an image tag that will need to be put here: https://github.com/osmosis-labs/osmosis/blob/dd1d910d5a0fc4a81050690e7d0e84158faebc3d/tests/e2e/containers/config.go#L30

We do this to be able to build genesis and config files from v14.x to start up Osmosis containers of that version and perform the upgrade to the next (v15) version in your branch.